ui/src/app/pages/admin/admin-institution-edit.tsx

```diff
@@ -646,27 +580,22 @@ export const AdminInstitutionEdit = fp.flow(withNavigation, withRouter)(class ex
         </SemiBoldHeader>
         <hr style={{border: '1px solid #A9B6CB'}}/>
         <FlexRow style={{gap: '2rem'}}>
+          {tiers.map(accessTierShortName =>
           <TierConfig
+              key={accessTierShortName}
+              accessTierShortName={accessTierShortName}
               institution={institution}
-              accessTierShortName={AccessTierShortNames.Registered}
-              TierBadge={() => <RegisteredTierBadge style={styles.tierBadge}/>}
-              setEraRequired={(value) => this.setRtRequireEra(value)}
-              setTierRequirement={(requirement) => this.setRegisteredTierRequirement(requirement)}
-              filterEmptyAddresses={() => this.filterEmptyRtEmailAddresses()}
-              setTierAddresses={(addrs) => this.setRegisteredTierEmails(addrs)}
-              filterEmptyDomains={() => this.filterEmptyRtEmailDomains()}
-              setTierDomains={(domains) => this.setRegisteredTierDomains(domains)}/>
-          <TierConfig
-              institution={institution}
-              accessTierShortName={AccessTierShortNames.Controlled}
-              TierBadge={() => <ControlledTierBadge style={styles.tierBadge}/>}
-              setEnableControlledTier={(value) => this.setEnableControlledTier(value)}
-              setEraRequired={(value) => this.setCtRequireEra(value)}
-              setTierRequirement={(requirement) => this.setControlledTierRequirement(requirement)}
-              filterEmptyAddresses={() => this.filterEmptyCtEmailAddresses()}
-              setTierAddresses={(addrs) => this.setControlledTierEmails(addrs)}
-              filterEmptyDomains={() => this.filterEmptyCtEmailDomains()}
-              setTierDomains={(domains) => this.setControlledTierDomains(domains)}/>
+              TierBadge={() => switchCase(accessTierShortName,
+                [AccessTierShortNames.Registered, () => <RegisteredTierBadge style={styles.tierBadge}/>],
+                [AccessTierShortNames.Controlled, () => <ControlledTierBadge style={styles.tierBadge}/>],
+                )}
+              setEnableControlledTier={(value) => this.setEnableControlledTier(accessTierShortName, value)}
```

Comment:
Note: RT does not have a way to trigger this method.  TierConfig only enables this control for CT.